Bottom: 068d5afc22029722bb5add0fd40b1448f0b8e651
Top:    3038a0f29cb30b2ae45da91054dec79d6bc2e21a
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-12-19 01:26:15 +0000

hid/gerber: Fixup trailing zeros from commit b90fb6ae22ae83fdc0c802268dcbe74005bc705e

(The trailing zero needs to be outside the format specifier I think)?


---

diff --git a/src/hid/gerber/gerber.c b/src/hid/gerber/gerber.c
index 97a7c29..3973814 100644
--- a/src/hid/gerber/gerber.c
+++ b/src/hid/gerber/gerber.c
@@ -672,7 +672,7 @@ gerber_set_layer (const char *name, int group, int empty)
 	    }
           /* Notice the last zeroes are literal zeroes here, a  *
            *  x10 scale factor.  v        v                     */
-          pcb_fprintf (f, "X%06.00mlY%06.00ml\r\n",
+          pcb_fprintf (f, "X%06.0ml0Y%06.0ml0\r\n",
 		   gerberDrX (PCB, pending_drills[i].x),
 		   gerberDrY (PCB, pending_drills[i].y));
 	}
